### ISSUE TITLE
Not work very well with kubernetes 1.16 and istio 1.4.3 deployed via rancher[BUG]

### ISSUE THREAD

**Issue by LaysDragon** [istio-ecosystem/admiral#65]
**Describe the bug**
After setup the sync step by step from README,the connection between cluster not working.
curl only get `503 (upstream connect error or disconnect/reset before headers)`.
If I trying to restart the gatewayingress pod,the gateway will stuck at
`Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?)`
and won't start up successes until I remove those ServiceEntry config created by Admiral,it looks like some kind of config conflicted or broken,but I can't find any useful log from other istio component. This problem occurred on my two cluster at the same time.
The Istio mesh spanning multiple cluster can work if I trying to setup manually via istio office document.
In the end i can only remove admiral and remove those auto-craft configs and use the manually  config way to setup those service. Still looking forward to further development of Admiral, this project looks great. 

other notes:
Admiral will crash several times while startup,sorry I forget to save the log. Might related to kubernetes version? I wondering what kubernetes version current Admiral developing on. It looks like Admiral not really work well with kubernetes 1.16.

**Steps To Reproduce**
Not sure if this can reproduce on similar environment
Cluster 1 rancher deployed  kubernetes 1.16.6 
Cluster 2 azure aks 1.16.4
All with rancher istio 1.4.3
Deployed sample from README

**Expected behavior**
Sample deployed should work as excepted.

**Comment by aattuluri** [comment]
@LaysDragon We have definitely not tested it with k8s 1.16. A todo item.

Admiral was tested with k8s 1.14 (max version) and Istio 1.4.3 (max version).

This was great feedback, we will verify this behavior and close the issue once we fix any issue.

**Comment by LaysDragon** [comment]
Forget the mention about the crash part,I remember it will crash with nil pointer at https://github.com/istio-ecosystem/admiral/blob/master/admiral/pkg/clusters/serviceentry.go

**Comment by palimarium** [comment]
Hi @aattuluri 
Getting the same issue with k8s  v.1.17.2

Logs: https://gist.github.com/palimarium/6b245f64eafef92553fe1dc86cb74384

**Comment by vivekpd15** [comment]
Similar issue with Istio 1.5.0 and Kuberenetes 1.15.2

**Comment by aattuluri** [comment]
@palimarium @vivekpd15 Thanks for reporting it. We will have Istio 1.5 and k8s 1.15.x support in v0.9 release scheduled for end of this month.

@vivekpd15 It would help if you can share logs if you saved any.

**Comment by LaysDragon** [comment]
I feeling that serviceentry seems very easy to break something,I mean ....something is stuck rather than broken,so there is no error log while serviceentry cause ingressgateway stuck at not ready status.
Today I upgrade istio to 1.4.6 via rancher's helm,the ingressgateway stuck into not ready status again,everything looks fine but pilot just sync the CDS then do nothing.it looks like pilot is waiting something I don't known.
```
NAME                                                    CDS        LDS          EDS          RDS          PILOT                            VERSION
istio-ingressgateway-7bf68948f5-nrcdb.istio-system      SYNCED     NOT SENT     NOT SENT     NOT SENT     istio-pilot-7b6b78bb55-wd58p     1.4.6
```
I spent the whole afternoon and finally fix it by remove manually created serviceentry before.
It will happen if readd the serviceentry back.
it might related to https://github.com/istio/istio/issues/21348 I guess,some looooong delay problem that I not I didn't figure out what just happen for some specified network condition.

**Comment by aattuluri** [comment]
The `latest` tag works for k8s 1.18.0 and Istio 1.5.1. It will be released as part of `v0.9`
